src/migrations/20191112101055-create-user.js

```diff
@@ -7,28 +7,81 @@ module.exports = {
       type: Sequelize.INTEGER
     },
     firstName: {
-      type: Sequelize.STRING
+      allowNull: false,
+      type: Sequelize.STRING,
     },
     lastName: {
-      type: Sequelize.STRING
-    },
-    email: {
-      type: Sequelize.STRING
+      allowNull: false,
+      type: Sequelize.STRING,
     },
     password: {
-      type: Sequelize.STRING
+      allowNull: false,
+      type: Sequelize.STRING,
+      defaultValue: 'password',
```

Comment:
Why do we have a default password value? I think this is insecure.

Reply:
If a user signs up with google, and tries to log in with his gmail and the default `password` as password, would the user be authenticated?